README.md

```diff
@@ -1 +1,106 @@
-# chaos-actions
+# Github Action for Chaos Engineering in Kubernetes
+
+This action provides a way to perform different chaos engineering experiments in the Kubernetes environment. It contains Litmus Chaos experiments to run the chaos and find the weakness in the system. For more details about chaos engineering in Kubernetes using Litmus visit <a href="https://docs.litmuschaos.io/docs/next/getstarted.html"> litmus-docs </a>.
+
+## Pre-requisites
+
+Kubernetes 1.11 or later.
+
+## Overview.
```

Comment:
Lets add reference to action usage example for each exp

Reply:
chaos action over "experiment". We are not providing users ability for users to specify exit checks